eng/Versions.props

```diff
@@ -11,7 +11,7 @@
     <MicrosoftNetFrameworkReferenceAssembliesVersion>1.0.0-alpha-004</MicrosoftNetFrameworkReferenceAssembliesVersion>
     <MicrosoftPrivateCoreFxNETCoreAppVersion>4.6.0-preview6.19229.1</MicrosoftPrivateCoreFxNETCoreAppVersion>
     <MicrosoftNETCorePlatformsVersion>3.0.0-preview6.19229.1</MicrosoftNETCorePlatformsVersion>
-    <MicrosoftNETCoreAppVersion>3.0.0-preview5-27620-01</MicrosoftNETCoreAppVersion>
+    <MicrosoftNETCoreAppVersion>3.0.0-preview6-27708-04</MicrosoftNETCoreAppVersion>
```

Comment:
I recommend separating the version properties coming from different repos to avoid as many merge conflicts here. For example, core-setup has:

```xml
    <!-- corefx -->
    <MicrosoftPrivateCoreFxNETCoreAppPackageVersion>4.6.0-preview6.19256.13</MicrosoftPrivateCoreFxNETCoreAppPackageVersion>
    <MicrosoftNETCorePlatformsPackageVersion>3.0.0-preview6.19256.13</MicrosoftNETCorePlatformsPackageVersion>
    <MicrosoftNETCoreTargetsPackageVersion>2.0.0</MicrosoftNETCoreTargetsPackageVersion>
    <!-- standard -->
    <NETStandardLibraryPackageVersion>2.1.0-prerelease.19230.1</NETStandardLibraryPackageVersion>
    <!-- coreclr -->
    <MicrosoftNETCoreRuntimeCoreCLRPackageVersion>3.0.0-preview6-27706-72</MicrosoftNETCoreRuntimeCoreCLRPackageVersion>
```

(If adjacent lines changes, that's a merge conflict. Blank lines don't work because Maestro++ removes them.)

Reply:
Is there a list of which dependencies belong to which repo so I can add the comments.  I'd like to avoid these merge conflicts in the future since it is a pain...